.github/workflows/build.yml

```diff
@@ -25,26 +25,41 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: movies
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
     strategy:
       fail-fast: false
       matrix:
         include:
           - scala: 2.12.17
             scalaShort: "2.12"
-            overall: 0.0
+            overall: 75.0
             changed: 80.0
           - scala: 2.13.12
             scalaShort: "2.13"
-            overall: 0.0
+            overall: 75.0
```

Comment:
ha, I didn't know that it's already this high :) cool, why not 👍

Reply:
This can be done by team decision.